File: backend-csharp/Models/PhieuKiemKho.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class PhieuKiemKho
{
    public long Id { get; set; }

    public string MaPhieu { get; set; } = null!;

    public long? NguoiTao { get; set; }

    public string? TrangThai { get; set; }

    public string? GhiChu { get; set; }

    public int? TongChenhLech { get; set; }

    public int? LenhTang { get; set; }

    public int? LenhGiam { get; set; }

    public DateTimeOffset CreatedAt { get; set; }

    public DateTimeOffset UpdatedAt { get; set; }

    public virtual ICollection<ChiTietKiemKho> ChiTietKiemKhos { get; set; } = new List<ChiTietKiemKho>();

    public virtual TaiKhoan? NguoiTaoNavigation { get; set; }
}

File: backend/models/PhieuKiemKho.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const PhieuKiemKho = sequelize.define(
  "PhieuKiemKho",
  {
    id: { type: DataTypes.BIGINT, primaryKey: true, autoIncrement: true },
    ma_phieu: { type: DataTypes.STRING(20), allowNull: false, unique: true },
    nguoi_tao: { type: DataTypes.BIGINT, allowNull: true },
    trang_thai: { type: DataTypes.STRING(20), defaultValue: "balanced" }, // balanced | cancelled
    ghi_chu: { type: DataTypes.TEXT, allowNull: true },
    tong_chenh_lech: { type: DataTypes.INTEGER, defaultValue: 0 },
    lenh_tang: { type: DataTypes.INTEGER, defaultValue: 0 },
    lenh_giam: { type: DataTypes.INTEGER, defaultValue: 0 },
  },
  {
    tableName: "PhieuKiemKho",
    timestamps: true,
    createdAt: "created_at",
    updatedAt: "updated_at",
  }
);

module.exports = PhieuKiemKho;
